apps/src/music/views/MusicView.jsx

```diff
@@ -109,6 +111,7 @@ class UnconnectedMusicView extends React.Component {
     clearPlaybackEvents: PropTypes.func,
     addPlaybackEvents: PropTypes.func,
     currentlyPlayingBlockIds: PropTypes.array,
+    isHeadersShowing: PropTypes.bool,
```

Comment:
I can't help feeling like this needs to be "areHeadersShowing" but "is" is better naming for a boolean value 😆

Reply:
I had the exact same dilemma!